rita_client/src/dashboard/neighbors.rs

```diff
@@ -61,37 +60,30 @@ pub fn get_neighbor_info(
     _req: HttpRequest,
 ) -> Box<dyn Future<Item = Json<Vec<NodeInfo>>, Error = RitaClientError>> {
     let debts = dump();
+    let neighbors = tm_get_neighbors().unwrap();
+    let combined_list = merge_debts_and_neighbors(neighbors, debts);
+    let babel_port = settings::get_rita_client().network.babel_port;
+
     Box::new(
-        TunnelManager::from_registry()
```

Comment:
this futures chain can be replaced using new babel_monitor as well.

Reply:
fixed